setup.py

```diff
@@ -52,6 +52,6 @@ def load_requirements(*requirements_paths):
         'Programming Language :: Python',
     ],
     packages=PACKAGES,
-    install_requires=load_requirements('requirements.txt'),
+    install_requires=load_requirements('requirements.txt', 'tox-requirements.txt'),
```

Comment:
This Django-1.11-upgrade PR demonstrates how to avoid including a `tox-requirements.txt` file in setup.py, since setup.py should be tox-agnostic:
https://github.com/edx/event-tracking/pull/38